tamr_client/schema_mapping/attribute_mapping.py

```diff
@@ -0,0 +1,189 @@
+"""
+See https://docs.tamr.com/new/reference/retrieve-projects-mappings
+"""
+from copy import deepcopy
+from typing import Dict, List, Optional, Tuple
+
+from tamr_client import attribute, dataset, project, response
+from tamr_client._types import (
+    Attribute,
+    AttributeMapping,
+    Instance,
+    JsonDict,
+    Project,
+    Session,
+    URL,
+)
+from tamr_client.exception import TamrClientException
+
+
+class NotFound(TamrClientException):
+    """Raised when referencing an attribute mapping that does not exist on the server."""
+
+    pass
+
+
+class AlreadyExists(TamrClientException):
+    """Raised when an attribute mapping with these specifications already exists."""
+
+    pass
+
+
+class Ambiguous(TamrClientException):
+    """Raised when an attribute mapping specification is incomplete, ambiguous, or contradictory."""
+
+    pass
+
+
+def create(
+    session: Session,
+    tamr_project: Project,
```

Comment:
Here there is an issue in that `project` has already been defined in
```python
from tamr_client import attribute, dataset, project, response
```
Do you think it is better practice to call this variable something like `proj`?  ADR 8 prescribes that this method of importing the module `project` is the correct one.

Reply:
The function argument name will be user-facing whereas the name for the `project` module within the `attribute_mapping` module will not be user-facing. So that would point me towards aliasing the `import` for `project` to something else to avoid the name collision

Reply:
That's good reasoning.  I've gone with the alias `tc_project` to mirror the `tc.project` we use in scripts.  I'm open to any other suggestions.